internal/process/defunct_processes.go

```diff
@@ -0,0 +1,91 @@
+package process
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+// ProcessFS is the process file system.
+const ProcessFS = "/proc"
+
+// Stat represents status information of a process from /proc/[pid]/stat.
+type Stat struct {
+	// Comm is the command name (usually the executable filename).
+	Comm string
+
+	// State is the state of the process.
+	State string
+}
+
+// DefunctProcesses returns the number of zombie processes in the node.
+func DefunctProcesses() (defunctCount uint, retErr error) {
+	directories, err := os.Open(ProcessFS)
+	if err != nil {
+		return 0, err
+	}
+	defer directories.Close()
+
+	names, err := directories.Readdirnames(-1)
+	if err != nil {
+		return 0, err
+	}
+
+	for _, name := range names {
+		// Processes have numeric names. If the name does not start with a number, skip.
+		if name[0] < '0' || name[0] > '9' {
+			continue
+		}
+
+		// If the name cannot be parsed to an int, it is not numeric.
+		_, err := strconv.ParseInt(name, 10, 0)
+		if err != nil {
+			continue
+		}
+
+		stat, err := status(name)
+		if err != nil {
+			logrus.Warnf("Failed to get the status of process with PID %s: %v", name, err)
```

Comment:
Let's move the verbosity down to debug
```suggestion
			logrus.Debugf("Failed to get the status of process with PID %s: %v", name, err)
```